qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -198,27 +194,29 @@ POST /collections/{collection_name}/cluster
 }
 ```
 
-After the transfer is initiated, the service will keep both copies of the shard updated until the transfer is complete.
-It will also make sure the transferred shard indexing process is keeping up before performing a final switch. This way, Qdrant ensures that there will be no degradation in performance at the end of the transfer.
+After the transfer is initiated, the service will keep both copies of the shard in sync until the transfer is complete.
+It will also make sure the transferred shard indexing process is keeping up before performing a final switch. This way, Qdrant ensures that there will be no degradation in performance at the end of the transfer. Once the transfer is completed, the old shard is deleted from the original node.
 
-In case you want to downscale the cluster, you can move all shards away from a peer and then remove the peer using [Remove peer from the cluster API](https://qdrant.github.io/qdrant/redoc/index.html#tag/cluster/operation/remove_peer).
+In case you want to downscale the cluster, you can move all shards away from a peer and then remove the peer using the [remove peer API](https://qdrant.github.io/qdrant/redoc/index.html#tag/cluster/operation/remove_peer).
 
 ```http
 DELETE /cluster/peer/{peer_id}
 ```
 
-After that, Qdrant will exclude the node from the consensus, and the instance will be ready for the shutdown.
+After that, Qdrant will exclude the node from the consensus, and the instance will be ready for shutdown.
 
 ## Replication
 
-*As of v0.11.0:* Qdrant allows to replicate shards between nodes in the cluster.
+*As of v0.11.0:* Qdrant allows you to replicate shards between nodes in the cluster.
+
+Shard replication increases the reliability of the cluster by keeping several copies of a shard spread across the cluster.
+This ensures the availability of the data in case of node failures, except if all replicas are lost.
 
-Shard replication increases the reliability of the cluster by keeping several copies of a shard spread among the cluster.
-This ensure the availability of the shards in case of node failures, except if all replicas are lost.
+### Replication factor
 
-By default, all the shards in a cluster have a replication factor of one, meaning no additional copy is maintained.
+When you create a collection, you can control how many shard replicas you'd like to store by changing the `replication_factor`. By default, `replication_factor` is set to "1", meaning no additional copy is maintained automatically. You can change that by setting the `replication_factor` when you create a collection.
 
-The replication factor of a collection can be configured at creation time.
+The replication factor of a collection can only be configured at creation time.
```

Comment:
I think, here we need more detail. In fact, you can change the replication_factor at runtime via the PATCH collection API, and it does reflect the change when fetching the collection config afterwards. However it has no effect on the actual amount of shard replicas in the running collection. Users need to adjust the actual replicas to the new replication factor themselves. Or is it better to deny such an update in the API to avoid confusion for now? WDYT @generall @agourlay @timvisee

Reply:
Yeah it is probably best to reject updates that change this parameter now.

Eventually we do want to support this though.

Reply:
I reworded to "Currently, the replication factor of a collection can only be configured at creation time" to leave room for the future 👍